compiler/passes/convert-uast.cpp

```diff
@@ -281,8 +290,60 @@ struct Converter {
     return nullptr;
   }
 
+  Expr* convertScanReduceOp(UniqueString op) {
+    if (op == "+") return new UnresolvedSymExpr("SumReduceScanOp");
```

Comment:
Arguably these should be using cached unique strings. What is here ends up being a strcmp. However I think they both more or less amount to the same thing. I'm curious which would be faster in practice. I certainly think the `op == "+"` style is clearer, if it is acceptable to us. If that also happens to be faster / about as fast, maybe we don't every need to cache unique strings (`atomicStr` etc).